src/controller/python/test/Dockerfile

```diff
@@ -0,0 +1,16 @@
+
+FROM ubuntu:focal
+
+ENV DEBIAN_FRONTEND noninteractive
+
+COPY /whl/python/*.whl /whl/
+COPY entrypoint.sh /entrypoint.sh
+COPY mobile-device-test.py /usr/bin/
+
+RUN apt-get update && \
+    apt-get install -y ca-certificates libglib2.0-dev avahi-daemon libavahi-client3 \
```

Comment:
Consider using  `install --no-install-recommends` to only install necessary packages.